README.md

```diff
@@ -24,43 +24,120 @@ To keep the code base and the API simple, zerolog focuses on JSON logging only.
 * `net/http` helpers
 * Pretty logging for development
 
-## Usage
-
+## Installation
 ```go
-import "github.com/rs/zerolog/log"
+go get -u github.com/rs/zerolog/log
 ```
+## Getting Started
+### Simple Logging
+For simple logging, import the global logger package **github.co/rs/zerolog/log**
+```go
+package main
 
-### A global logger can be use for simple logging
+import (
+	"github.com/rs/zerolog/log"
+)
 
-```go
-log.Print("hello world")
+func main() {
+	// I like UNIX Time - if you set zerolog.TimeFieldFormat to an
+	// empty string, logs will write with UNIX time
+	zerolog.TimeFieldFormat = ""
+
+	log.Print("hello world")
+}
 
-// Output: {"level":"debug","time":1494567715,"message":"hello world"}
+// Output: {"time":1516134303,"level":"debug","message":"hello world"}
 ```
+> Note: The default logging level for log.Print is *debug*
+----
+### Leveled Logging
 
+#### Simple Leveled Logging Example
 
 ```go
-log.Info().Msg("hello world")
+package main
+
+import (
+	"github.com/rs/zerolog/log"
+)
+
+func main() {
+	log.Info().Msg("hello world")
 
-// Output: {"level":"info","time":1494567715,"message":"hello world"}
+}
+
+// Output: {"time":1516134303,"level":"info","message":"hello world"}
 ```
 
-NOTE: To import the global logger, import the `log` subpackage `github.com/rs/zerolog/log`.
+**zerolog** allows for logging at the following levels (from highest to lowest):
+- panic (PanicLevel, 5)
+- fatal (FatalLevel, 4)
+- error (ErrorLevel, 3)
+- warn (WarnLevel, 2)
+- info (InfoLevel, 1)
+- debug (DebugLevel, 0)
 
+You can set the Global logging level to any of these options using the SetGlobalLevel function in the zerolog package, passing in one of the given constants above, e.g. zerolog.InfoLevel would be the "info" level.  Whichever level is chosen, all logs with a level greater than or equal to that level will be written. To turn off logging entirely, pass the Disabled constant.
```

Comment:
Could you please backquote "code" like (`SetGlobalLevel` and `zerolog.InfoLevel`) in the text?

Reply:
Yes, will do